art/detection/subsetscanning/scoring_functions.py

```diff
@@ -0,0 +1,71 @@
+""" Scanner scoring functions """
+
+import numpy as np
+
+
+class ScoringFunctions:
+    """ Scanner scoring functions
+        These functions are used in the scanner to determine the score of a subset
+    """
+
+    @staticmethod
+    def get_score_bj_fast(n_alpha, no_records, alpha):
```

Comment:
Could you please add a docstring describing what the function does, its arguments and return similar to the other function docstrings in ART?